package.json

```diff
@@ -3,7 +3,7 @@
     "displayName": "wsdl2rest",
     "description": "A VS Code extension supporting wsdl2rest via Camel Rest DSL",
     "license": "Apache-2.0",
-    "version": "0.0.4",
+    "version": "0.0.5",
```

Comment:
I'm expecting the package-lock.json to be updated too then.
Apart from that, I think it is a good practice to update the version in a separate PR as soon as the previous release has been published.

Reply:
Updated package-lock